migrations.lock

```diff
@@ -147,6 +147,9 @@ blobs
  0009_delete_blobexpiration
  0010_auto_20191023_0938
  0011_blobmeta_compressed
+captcha
+ 0001_initial
+ 0002_delete_captchastore
```

Comment:
This is surprising since you didn't change anything with captcha. Do you know why this was added? Maybe you need to updated requirements in your virtualenv?

Reply:
this happened to me too, and yes, updating requirements is needed here.

rather than doing a `pip-sync` which destroyed my `virtualenv`, I recommend:
```
pip uninstall django-simple-captcha
pip install -r requirements/requirements.txt
pip install -r requirements/dev-requirements.txt
```

You also might need to manually edit `migrations.lock` to remove these lines.